airflow/www/api/experimental/endpoints.py

```diff
@@ -41,6 +47,124 @@
 api_experimental = Blueprint('api_experimental', __name__)
 
 
+@api_experimental.route(
+    '/dags/<string:dag_id>/dag_runs/<string:execution_date>/tasks/<string:task_id>/logs',
+    methods=['GET'])
+@requires_authentication
+def logs(dag_id, execution_date, task_id):
+    """
+    Return logs for the specified task identified by dag_id, execution_date and task_id
+    """
+    try:
+        log = get_task_logs(dag_id, task_id, execution_date)
+    except AirflowException as err:
+        _log.info(err)
+        response = jsonify(error="{}".format(err))
+        response.status_code = err.status_code
+        return response
+    except ValueError:
```

Comment:
This error handling is overly broad and could catch a ValueError from something else.

Additionally: I would like to see the conversion from string to dt object here in this FN, not in side the common api part.

Reply:
done.